inbox/models/category.py

```diff
@@ -18,6 +19,36 @@
 EPOCH = datetime.utcfromtimestamp(0)
 
 
+def sanitize_name(name):
+    # g_label may not have unicode type (in particular for a numeric
+    # label, e.g. '42'), so coerce to unicode.
+    name = unicode(name)
+    # Remove trailing whitespace, truncate (due to MySQL limitations).
+    name = name.rstrip()
+    if len(name) > MAX_INDEXABLE_LENGTH:
+        name = name[:MAX_INDEXABLE_LENGTH]
+    return name
+
+
+class CategoryNameString(StringWithTransform):
```

Comment:
I wonder if this extra subclass is still useful? I actually feel like having the StringWithTransform(Sanitize_name) followed by the sanitize_name validation in the same file might be easiest to understand? Totally up to you!